src/main/java/minicraft/saveload/LegacyLoad.java

```diff
@@ -104,7 +104,6 @@ public void loadFromFile(String filename) {
 			br = new BufferedReader(new FileReader(filename));
 			
 			String curLine;StringBuilder total = new StringBuilder();
-			ArrayList<String> curData;
```

Comment:
Please don't touch LegacyLoad. It is called that for a reason.

Reply:
Fair, I will remove this